### DISCUSSION TITLE
How to check which plot won the block?

### DISCUSSION THREAD

**Discussion by spik3yp** [Chia-Network/chia-blockchain#6914] [Support]
I have tried to look up the proof_of_space.plot_public_key from the chiaexplorer and search the key on my Chia GUI (windows OS)> Plots tab. However, I can't find the plot. Any help will be appreciated. 
Thanks.

**Comment by rmica** [comment]
I guess that you have to pick the right block. Try to search the height when you farmed the block. The next height peak after the farm. You can look into the logs or with the CLI "chia farm summary" ("Last height farmed").

**Reply by spik3yp** [reply]
Thanks for your reply.
I have managed to find it with your help.
To sum up what I did:
1. Go to Chia GUI > Plots
2. Copy your "Pool Key"
3. Go to https://www.chiaexplorer.com/
4. Search using your "Pool Key"
5. It should show your winning blocks with the "height"
6. Click on the "height" to go to the detail page
7. Search for "proof_of_space.plot_public_key"
8. Now, this is your plot public key that you can find in your Chia GUI > Plots tab

Hope it will help someone in the future.

Thanks.

**Reply by rmica** [reply]
Good! Thanks to you too.

**Reply by cjedwards** [reply]
so what happens when there are multiple block winners?

**Reply by mstfksknklc** [reply]
Thanks
